Zend/zend_API.c

```diff
@@ -2928,6 +2953,27 @@ ZEND_API zend_result zend_register_functions(zend_class_entry *scope, const zend
 			zend_string_release(lowercase_name);
 			break;
 		}
+		if (reg_function->frameless_function_infos) {
+			const zend_frameless_function_info *flf_info = reg_function->frameless_function_infos;
+			while (flf_info->handler) {
+				if (zend_flf_count == zend_flf_capacity) {
+					if (!zend_flf_capacity) {
+						zend_flf_capacity = 8;
+					} else {
+						zend_flf_capacity *= 2;
+					}
+					/* +1 for NULL terminator */
+					zend_flf_handlers = realloc(zend_flf_handlers, (zend_flf_capacity + 1) * sizeof(void *));
+					zend_flf_functions = realloc(zend_flf_functions, (zend_flf_capacity + 1) * sizeof(zend_function *));
+				}
+				zend_flf_handlers[zend_flf_count] = flf_info->handler;
+				zend_flf_functions[zend_flf_count] = (zend_function *)reg_function;
+				zend_flf_count++;
+				zend_flf_handlers[zend_flf_count] = NULL;
```

Comment:
This likely won't matter, but you could move this NULLing out of the loop (if you wrap that and the loop in an if condition, and change the loop to do-while to avoid a needless initial check).